custom-vu/src/main/kotlin/jces1209/vu/action/ProjectIssueNavigatorAction.kt

```diff
@@ -0,0 +1,31 @@
+package jces1209.vu.action
+
+import com.atlassian.performance.tools.jiraactions.api.BROWSE_PROJECTS
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.ProjectMemory
+import jces1209.vu.page.project.ProjectIssueNavigatorPage
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+
+class ProjectIssueNavigatorAction(
+    private val jira: WebJira,
+    private val meter: ActionMeter,
+    private val projectKeyMemory: ProjectMemory
+
+) : Action {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    override fun run() {
+        val projectKey = projectKeyMemory.recall()
+        if (projectKey == null) {
+            logger.debug("I don't recall any issue keys. Maybe next time I will.")
+            return
+        }
+        meter.measure(BROWSE_PROJECTS) {
+            jira.goToProjectSummary(projectKey.key)
```

Comment:
also, it's necessary to add `wait for page load` - FalliblePage should help with it
or use the new page classes